xtask/src/config.rs

```diff
@@ -0,0 +1,63 @@
+use common::{make_path, path::PathsError, PathBuf};
+use normpath::BasePathBuf;
+use once_cell::sync::Lazy;
+
+pub static PATHS: Lazy<Paths> = Lazy::new(Paths::new);
+
+pub struct Paths {
```

Comment:
it seems like more a config, maybe `Config` would b a more appropriate name for this